src/cli/cli.cpp

```diff
@@ -806,6 +806,9 @@ otError Interpreter::ProcessChild(uint8_t aArgsLength, char *aArgs[])
                 OutputFormat("|%1d", childInfo.mRxOnWhenIdle);
                 OutputFormat("|%1d", childInfo.mFullThreadDevice);
                 OutputFormat("|%1d", childInfo.mFullNetworkData);
+                OutputFormat("| %1d ", childInfo.mVersion);
+                OutputFormat("| %1d ", childInfo.mIsCslSynced);
+                OutputFormat("| %5d ", childInfo.mQueuedMessageCnt);
```

Comment:
```suggestion
                OutputFormat("|%1d ", childInfo.mVersion);
                OutputFormat("|%1d ", childInfo.mIsCslSynced);
                OutputFormat("|%5d ", childInfo.mQueuedMessageCnt);
```

Reply:
The whitespace is used to fill up the unused width in the table like RLOC16 above.